http_client_test.go

```diff
@@ -332,6 +332,42 @@ func TestHTTPClientGetReturnsErrorOn5xxFailure(t *testing.T) {
 	assert.Equal(t, "server error: 500", err.Error())
 }
 
+type myHTTPClient struct {
+	client http.Client
+}
+
+func (c *myHTTPClient) Do(request *http.Request) (*http.Response, error) {
+	request.Header.Set("foo", "bar")
+	response, err := c.client.Do(request)
```

Comment:
You can just do `return c.client.Do(request)`

Reply:
yup, made the change 👍